src/deepsparse/server/system_logging.py

```diff
@@ -32,13 +65,21 @@ def log_resource_utilization(pipeline: Pipeline, **kwargs: Any):
     pass
 
 
-def log_request_details(pipeline: Pipeline, **kwargs: Any):
+def log_request_details(
+    server_logger: BaseLogger,
+    prefix: str = REQUEST_DETAILS_IDENTIFIER_PREFIX,
+    **kwargs: Any,
```

Comment:
nit: rename to `**items` or something else more specific to indicate that these are actually the items to log